website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -125,6 +127,7 @@ The following examples create a materialized view:
 {{
   config(
     materialized = 'materialized_view',
+    auto_refresh = false,
```

Comment:
I believe this is a capital F; it's not the jinja `false`, it's the python `False`.

We're also adding the option to pass in case-insensitive strings, e.g. `"False"`, `"false"`, `"FALSE"`, etc. It's not there yet, and we can submit another PR that goes in at the same time if that makes more sense.

Reply:
I hate that but glad that's getting fixed :)